packages/docker/src/wait.coffee.md

```diff
@@ -39,7 +39,7 @@ console.info(`Did we really had to wait: ${status}`)
 
 ## Handler
 
-    handler = ({config, tools: {find, log}}) ->
+    handler = ({config, tools: {log}}) ->
```

Comment:
This shall be in a different commit, you are polluting the reviewing process and make it less relevant